include/mxnet/c_api.h

```diff
@@ -321,13 +321,18 @@ MXNET_DLL int MXDumpProcessProfile(int finished, int profile_process, KVStoreHan
 MXNET_DLL int MXDumpProfile(int finished);
 
 /*!
- * \brief Print aggregate stats to the a string
- * \param out_str Will receive a pointer to the output string
- * \param reset Clear the aggregate stats after printing
- * \return 0 when success, -1 when failure happens.
- * \note
- */
-MXNET_DLL int MXAggregateProfileStatsPrint(const char **out_str, int reset);
+	 * \brief Print aggregate stats to the a string
+	 * \param out_str Will receive a pointer to the output string
+	 * \param reset Clear the aggregate stats after printing
+	 * \param out_str will receive a pointer to the output string
+	 * \param reset clear the aggregate stats after printing
+	 * \param format whether to return in tabluar or json format
+	 * \param sort_by sort by avg, min, max, or count
+	 * \param ascending whether to sort ascendingly
+	 * \return 0 when success, -1 when failure happens.
+	 * \note
+	 */
+MXNET_DLL int MXAggregateProfileStatsPrint(const char **out_str, int reset, int format, int sort_by, int ascending);
```

Comment:
This is a breaking change. Please dont touch the existing API, introduce a new API MXAggregateProfileStatsPrintEx and add your functionality there.

Reply:
Yeah will do!